### PR TITLE
Bump color-rgba 2.1.1 - module color-normalize

### PR DESCRIPTION
**Pull request by archmoj**
Include fix for https://github.com/colorjs/color-parse/issues/1
@dy 
cc: @etpinard

### CODE REVIEW

**Comment by archmoj** [comment]
It seems that one test is not compatible with current "stable" node version 12.3.1.
The previous successful run on a "stable" version was performed on node version 10.5.0: https://travis-ci.org/colorjs/color-normalize/jobs/395637825.

**Comment by etpinard** [comment]
@archmoj do you have an idea why tests are failing in node 12?

**Comment by archmoj** [comment]
The issue was with `node 12` & not with `color-rgba 2.1.1` as reported here: https://github.com/colorjs/color-normalize/issues/3 (tests are failed even on the master branch when using node 12)!

The test is fixed to work with node 12.
@dy would you mind having the second look at this PR?
Thanks.

**Comment by archmoj** [comment]
@dy could you possibly release a patch from this?
Many thanks.

**Comment by zeke** [comment]
Looks good to me. I'd merge it, but I'm not an owner of the npm package.

**Comment by archmoj** [comment]
Thanks @zeke for the review.
Great news! 
Then we could wait for @dy for the npm package.

**Comment by archmoj** [comment]
@dy any chance we have a patch release this week? 
Thanks.

File: test.js [hunk 1]
```diff
@@ -1,14 +1,21 @@
 const assert = require('assert')
 const rgba = require('./')
 
+function assert_deepEqual(a, b) {
```
Comment (dy) [inline test.js:4]: @archmoj what was the cause for this method?

**Comment by dy** [comment]
Sorry for the delay. Done.

**Pull request merged**